variables.tf

```
@@ -1,111 +1,129 @@
   type        = list(string)
-  default     = []
 }
 
-variable "allowed_ipv6_cidr_blocks" {
-  description = "IPv6 CIDR blocks allowed to reach the database"
-  type        = list(string)
-  default     = []
+variable "backup_retention_period" {
+  default     = 5
+  description = "How long to keep RDS backups (in days)"
+  type        = string
 }
 
-variable "storage" {
-  description = "How much storage is available to the database"
-  type        = string
-  default     = 20
+variable "create_secretmanager_secret" {
+  default     = true
+  description = "True to create a secretmanager secret containing DB password"
+  type        = bool
 }
 
-variable "storage_type" {
-  description = "What storage backend to use (gp2 or standard. io1 not supported)"
-  type        = string
-  default     = "gp2"
+variable "create_ssm_secret" {
+  default     = false
+  description = "True to create a SSM Parameter SecretString containing DB password"
+  type        = bool
 }
 
 variable "engine" {
-  description = "Which RDS Engine to use"
-  type        = "string"
   default     = "postgres"
+  description = "Which RDS Engine to use"
+  type        = string
 }
 
 variable "engine_version" {
+  default     = "11.5"
   description = "Version of database engine to use"
   type        = string
-  default     = "11.5"
 }
 
 variable "instance_class" {
+  default     = "db.t3.small"
   description = "What instance size to use"
   type        = string
-  default     = "db.t3.small"
 }
 
 variable "multi_az" {
+  default     = true
   description = "whether to make database multi-az"
   type        = string
-  default     = true
 }
 
-variable "username" {
-  description = "username of master user"
+variable "name" {
+  default     = "postgres-rds"
+  description = "common name for resources in this module"
   type        = string
-  default     = "postgres_user"
 }
 
-variable "backup_retention_period" {
-  description = "How long to keep RDS backups (in days)"
+variable "pass_version" {
+  default     = 1
+  description = "Increment to force DB password change"
   type        = string
-  default     = 5
 }
 
 variable "port" {
+  default     = "5432"
   description = "Port the database should listen on"
   type        = string
-  default     = "5432"
 }
 
 variable "skip_final_snapshot" {
+  default     = false
   description = "If true no final snapshot will be taken on termination"
   type        = string
-  default     = false
 }
 
-variable "pass_version" {
-  description = "Increment to force DB password change"
+variable "storage" {
+  default     = 20
+  description = "How much storage is available to the database"
   type        = string
-  default     = 1
 }
 
-variable "allowed_password_chars" {
-  description = "What characters are allowed in the postgres password"
+variable "ssm_path" {
+  default     = ""
+  description = "Custom path for SSM parameter, only takes effect if create_ssm_secret is true. "
```

markdown makes you cool
```suggestion
  description = "Custom path for SSM parameter, only takes effect if `create_ssm_secret` is true. "
```